tensorflow_addons/image/filters.py

```diff
@@ -204,3 +200,87 @@ def median_filter2d(
         output = tf.cast(median, image.dtype)
         output = img_utils.from_4D_image(output, original_ndims)
         return output
+
+
+def gaussian_filter2d(
+    image: FloatTensorLike,
+    sigma: FloatTensorLike,
+    filter_shape: int,
```

Comment:
Please match the input of the other filters. Integer or tuple so that users can specify non-square kernels

Reply:
Addressed!